bench/irmin-pack/layers.ml

```diff
@@ -229,7 +229,7 @@ let first_5_cycles config repo =
   print_commit_stats config c 0 0.0;
   let rec aux i c =
     add_min c;
-    if i >= 4 then Lwt.return c
+    if i > 4 then Lwt.return c
```

Comment:
Only 4 cycles were running before, fixed to run 5 cycles as the function name suggests :smile_cat:

Reply:
One day, we'll live in an `Lwt` utopia with helper functions for these things...